server/views/privacy-notice.njk

```diff
@@ -0,0 +1,263 @@
+        </li>
+        <li>ask that the processing of your personal data is restricted in certain circumstances
+        </li>
+    </ul>
+
+    <p class="govuk-body">Contact the MoJ data protection officer if you wish to make a request:
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">MoJ Data Protection Officer
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">3rd Floor, Post Point 3.20
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">10 South Colonnades
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">London
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">E14 4PU
+    </p>
+    <a class="govuk-link" href="mailto:data.compliance@justice.gov.uk">data.compliance@justice.gov.uk
+    </a>
+
+    <p class="govuk-body govuk-!-margin-top-5">For more information on how and why your information is processed please
+        see the information
+        provided when you accessed our services or were contacted by us.</p>
+
+    <h2 class="govuk-heading-m">Links to other services
+    </h2>
+
+    <p class="govuk-body">The Prepare a case for sentence contains links to other services.</p>
+    <p class="govuk-body">This privacy policy only applies to Prepare a case for sentence and does not cover other
+        services and transactions that we link to. These services have their own terms and conditions and privacy
+        policies.
+    </p>
+
+    <h3 class="govuk-heading-s">Following a link to another service</h3>
+    <p class="govuk-body">If you come to Prepare a case for sentence from another service, we may receive information
+        from the other service. We do not use this data. You should read the privacy policy of the service you came from
+        to find out more about this.
+    </p>
+
+    <h3 class="govuk-heading-s">Following a link to the Prepare a case for sentence from another service</h3>
+    <p class="govuk-body">If you come to Prepare a case for sentence from another service, we may receive information
+        from the other service. We do not use this data. You should read the privacy policy of the service you came from
+        to find out more about this.</p>
+
+    <h2 class="govuk-heading-m">Complaints
+    </h2>
+
+    <p class="govuk-body">When we ask you for information, we will keep to the law. If you consider that your
+        information has been handled incorrectly, you can contact the MoJ data protection officer:
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">MoJ Data Protection Officer
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">3rd Floor, Post Point 3.20
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">10 South Colonnades
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">London
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">E14 4PU
+    </p>
+    <a class="govuk-link" href="mailto:data.compliance@justice.gov.uk">data.compliance@justice.gov.uk
```

Comment:
I'm not sure but does all the links should have the size of the text or should they be a little smaller?

Reply:
Yeah you're right I need to add the govuk-body class, good spot 🙂